pom.xml

```diff
@@ -322,6 +328,14 @@
             </reporting>
         </profile>
 
+        <profile>
+            <id>docker-toolbox</id>
+            <properties>
+                <!-- Define the ACS endpoint path when docker toolbox is used (docker container are not exposed through localhost) -->
+                <test.acs.endpoint.path>http://192.168.99.100:8080/alfresco</test.acs.endpoint.path>
```

Comment:
I assume this will only be an issue for Windows 7 / Docker toolbox? With Windows 10 / Docker for Desktop this won't have to be used?

I can't remember when properties are defined in a profile, can they still be modified from the commandline? I can easily imagine an IP like this could be different because someone have weird / custom settings in their docker setup. OK that probably didn't make sense, I guess my main question is: 
With this property in mind, can it be changed from the command line, for example like `mvn clean install -Pdocker-toolbox -Dtest.acs.endpoint.path=http://my.other.ip.address/alfresco` ?

Reply:
Perfect, just wanted to make sure :)